static-site/src/components/Footer/sitemap.jsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import styled from "styled-components";
+import Link from 'next/link'
+import { FaExternalLinkAlt } from "react-icons/fa";
+import * as Styles from "../splash/styles";
+import { BigSpacer } from "../../layouts/generalComponents";
+
+
+const UnstyledList = styled.ul`
+  list-style: none;
+`
+
+const ListItem = styled(Styles.FocusParagraph)`
+  line-height: 0.7;
```

Comment:
[subjective] I'd go for a line-height of 0.5, font-weight of 300, and color of `var(--text-color)` personally. What do you think?

Reply:
Added in eca5ef6b.

I already see an inherited font-weight of 300, but good to be explicit.

Not sure what you mean by `var(--text-color)` - nothing defines `--text-color` currently. Also, setting it here has no effect with this in place (which I could change):

https://github.com/nextstrain/nextstrain.org/blob/eca5ef6b8a5c525f0942369f7c1b69abb22fdb3c/static-site/src/components/Footer/sitemap.jsx#L87-L91

Reply:
>  Also, setting it here has no effect...

Ah, I didn't mean to say this was the correct component to add all of these styles, I was using it as a hook to describe what styles I thought we should end up with. Sorry for any confusion!

> I already see an inherited font-weight of 300, but good to be explicit.

As of 6b6f4520 I still get a font-weight of 500 in Firefox 125 via this definition:

https://github.com/nextstrain/nextstrain.org/blob/08db0bd73841c8e0ef798e679167fb11ae014664/static-site/src/components/layout.jsx#L39-L44

> Not sure what you mean by var(--text-color) - nothing defines --text-color currently

Not sure where I got that from. I'm sure it was set when I last tested but you're right I cannot find it anywhere. In fact I don't even think we set a page-default `color`! I believe our original intention was `#333`.

Reply:
> I still get a font-weight of 500

Oh you're right, I was looking in the wrong place. Fixed in b4727b1f4c3382019f7801df3b3e13f0913ffd06

> Ah, I didn't mean to say this was the correct component to add all of these styles, I was using it as a hook to describe what styles I thought we should end up with. Sorry for any confusion!
> I don't even think we set a page-default `color`! I believe our original intention was `#333`.

Gotcha, no worries! I'll defer this to an existing issue on global text colors: #863